<!-- YAML
added: v0.7.3
-->

此事件和 [`child_process.fork()`][]提供的error事件相同。

在一个工作进程中，可以使用`process.on('error')`

